model_zoo/easy_integrate_bundle.py

```diff
@@ -0,0 +1,248 @@
+                Const.KEY_DEVICE: device,
+            }
+
+            if multi_gpu:
+                config_paths = [
+                    c for c in Const.MULTI_GPU_CONFIGS if os.path.exists(os.path.join(self.bundle_path, "configs", c))
+                ]
+                if not config_paths:
+                    logger.warning(f"Ignore Multi-GPU Training; No multi-gpu train config {Const.MULTI_GPU_CONFIGS} exists")
+                    return
+
+                train_path = os.path.join(self.bundle_path, "configs", f"train_multigpu_fold{fold}.json")
+                multi_gpu_train_path = os.path.join(self.bundle_path, "configs", config_paths[0])
+                logging_file = os.path.join(self.bundle_path, "configs", "logging.conf")
+                for k, v in overrides.items():
+                    if k != Const.KEY_DEVICE:
+                        self.bundle_config.set(v, k)
+                ConfigParser.export_config_file(self.bundle_config.config, train_path, indent=2)
+
+                env = os.environ.copy()
+                env["CUDA_VISIBLE_DEVICES"] = ",".join([str(g) for g in gpus])
+                logger.info(f"Using CUDA_VISIBLE_DEVICES: {env['CUDA_VISIBLE_DEVICES']}")
+                cmd = [
+                    "torchrun",
+                    "--standalone",
+                    "--nnodes=1",
+                    f"--nproc_per_node={len(gpus)}",
+                    "-m",
+                    "monai.bundle",
+                    "run",
+                    "training",
+                    "--meta_file",
+                    self.bundle_metadata_path,
+                    "--config_file",
+                    f"['{train_path}','{multi_gpu_train_path}']",
+                    "--logging_file",
+                    logging_file,
+                ]
+                self.run_command(cmd, env)
+            else:
+                monai.bundle.run(
+                    "training",
+                    meta_file=self.bundle_metadata_path,
+                    config_file=self.bundle_config_path,
+                    **overrides,
+                )
+            fold += 1
+
+            logger.info(f"Fold{fold} Training Finished....")
+
+        if test_datalist is not None:
+            device = self._device(args.device)
+            self.ensemble_inference(device, test_datalist, ensemble=args.ensemble)
+
+    def run_command(self, cmd, env):
+        process = subprocess.Popen(cmd, stdout=subprocess.PIPE, universal_newlines=True, env=env)
+        while process.poll() is None:
+            line = process.stdout.readline()
+            line = line.rstrip()
+            if line:
+                print(line, flush=True)
+
+        logger.info(f"Return code: {process.returncode}")
+        process.stdout.close()
+
+
+if __name__ == '__main__':
+    """
+    Usage:
+        python easy_integrate_bundle.py --bundle_root /workspace/Code/Bundles/spleen_ct_segmentation --dataset_dir /workspace/Data/Task09_Spleen
```

Comment:
would be great to describe 
- the goal of this script
- the expected output
- the steps to achieve the goal (in more detail): 1. dataset splitting, 2. training multiple models using each split, 3, compute ensemble results using the models
- assumption of this script (how much GPU memory? which model zoo item? structure of the data list and dataset folders?)